src/main/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/readytolist/ReadyToListAboutToSubmitHandler.java

```diff
@@ -25,24 +37,48 @@ public boolean canHandle(CallbackType callbackType, Callback<SscsCaseData> callb
     }
 
     @Override
-    public PreSubmitCallbackResponse<SscsCaseData> handle(CallbackType callbackType, Callback<SscsCaseData> callback,
-                                                          String userAuthorisation) {
+    public PreSubmitCallbackResponse<SscsCaseData> handle(CallbackType callbackType, Callback<SscsCaseData> callback, String userAuthorisation) {
         if (!canHandle(callbackType, callback)) {
             throw new IllegalStateException("Cannot handle callback.");
         }
-        final CaseDetails<SscsCaseData> caseDetails = callback.getCaseDetails();
-        final SscsCaseData sscsCaseData = caseDetails.getCaseData();
 
-        PreSubmitCallbackResponse<SscsCaseData> callbackResponse = new PreSubmitCallbackResponse<>(sscsCaseData);
+        final CaseDetails<SscsCaseData> caseDetails = callback.getCaseDetails();
+        final SscsCaseData sscsCaseData = caseDetails.getCaseData(); //Maybe handle in each method instead of HEAD
+        if(schedulingAndListingFeature){
+            boolean listAssist = checkIfListAssist(sscsCaseData);
+            if(listAssist){
+                sscsCaseData.setHearingRoute("ListAssist");
```

Comment:
Generally using a plain string like this isn't best practice. It's a magic string hidden away in code and can easily be changed without any extra protection. If you use an enum, however, the compiler can help you. If someone was to change the enum value either all references to it have to change or the compiler will complain.